### ISSUE TITLE
Fix failed build in Circle when rebuilding without cache

### ISSUE THREAD

**Issue by lbeaufort** [18F/fec-cms#1746]
@LindsayYoung and I were attempting to rebuild in Circle without caching so we could make sure we had the most recent minor version updates for our dependences. When rebuilding the most recent deployment to the `dev` environment [resulted in the following error](https://circleci.com/gh/18F/fec-cms/670): 
```
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone --depth=1 -q -b master https://github.com/noahmanger/Accessible-Mega-Menu.git /root/.npm/_cacache/tmp/git-clone-41bf7874
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-41bf7874': Permission denied
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-02-08T17_54_30_617Z-debug.log
Exited with code 1
```

**Comment by patphongs** [comment]
@lbeaufort I think I've unblocked us with this PR ready for your review: https://github.com/18F/fec-cms/pull/1747

The issue was that `sudo npm install` was using root permissions to install. Due to the fact that circleCI runs as a normal user and not root, using `sudo npm install` causes permission issues. When using regular permissions, that allows circle to access the normal local circle user's `.npm` in order to write to it. 

Also while I was in SSH mode in the build, noticed some webpack configuration object errors Invalid configuration object (see below). Essentially the assetSort object does not exist, it needed to be `assetsSort` plural. This fixes our ability to run webpack 3.10.0 without issue. cc @LindsayYoung 

```Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration[0].stats should be one of these:
   object { all?, context?, hash?, version?, timings?, performance?, depth?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warnings?, errors?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? } | boolean | "none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose"```

**Comment by lbeaufort** [comment]
Addressed by https://github.com/18F/fec-cms/pull/1747
